ex03.ex

```diff
@@ -81,7 +84,8 @@ defmodule Ex03 do
 
   """
 
-  def list_contains . .. "your code"
+  def list_contains([], _num), do: false
```

Comment:
# -1

How about using pattern matching for the test?

~~~ elixir
  def list_contains([], _num), do: false
  def list_contains([ num | _t ], num), do: true
  def list_contains([ _h | t ], num), do: list_contains(t, val)
~~~

I'd argue it is simpler, as your line 88 really does two things (the equality test and then an `||`)